e2e/chat.spec.ts

```diff
@@ -164,4 +164,43 @@ describe('chat', () => {
 
     expect(formattedAmount).toBe(0.2345);
   });
+  test('model dropdown interactions', async ({ page }) => {
+    const NUMBER_OF_SUPPORTED_MODELS = 3;
+    const DEFAULT_MODEL_NAME = 'gpt-4o-mini';
```

Comment:
These will likely change, but at least there's only one place to update in these tests when that happens.

Reply:
Maybe we can move `DEFAULT_MODEL_NAME` into a config in the core directory. The AppContextProvider also sort of sets a "default" selected model and that is where I'd expect it to change.

If we change the default behaviour there, I would expect the e2e test to have to be updated as well. If we set this here we run the risk of having the test pass when the default changes in app context which has the potential to hide bugs.